### DISCUSSION TITLE
Possible RPM packaging bug?

### DISCUSSION THREAD

**Discussion by CompsProject** [atom/atom#23190] [Q&A]
yum install atom rpm gives an error on my centos7. It comes out during the dependency check. 
Here is the -v verbose output.  I *think* there may be a syntax problem in the check for libcurl.so 

Examining atom.x86_64.rpm: atom-1.58.0-0.1.x86_64
Marking atom.x86_64.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package atom.x86_64 0:1.58.0-0.1 will be installed
Checking deps for atom.x86_64 0:1.58.0-0.1 - u
looking for ('(glib2 or kde-cli-tools or xdg-utils)', None, (None, None, None)) as a requirement of atom.x86_64 0:1.58.0-0.1 - u
looking for ('(libcurl.so.3()(64bit) or libcurl.so.4()(64bit))', None, (None, None, None)) as a requirement of atom.x86_64 0:1.58.0-0.1 - u
Error: Invalid version flag: or

**Comment by ferdnyc** [comment]
@CompsProject The package is using [RPM boolean dependencies](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/packaging_and_distributing_software/new-features-in-rhel-8_packaging-and-distributing-software#support-for-boolean-dependencies_new-features-in-rhel-8), which require RPM 4.13+ (and may not be supported by any version of `yum`, though I'm not certain about that).

So, not a syntax _error_ per se, just that the RPM as currently packaged requires Fedora 27+ or RHEL/CentOS 8.
